modules/460-log-shipper/templates/rbac-for-us.yaml

```diff
@@ -33,6 +33,7 @@ rules:
     resources:
       - namespaces
       - pods
+      - nodes
```

Comment:
I'm just curious why do we need it?

Reply:
https://vector.dev/highlights/2022-07-07-0-23-0-upgrade-guide/#kubernetes-logs-list-watch-nodes
https://github.com/vectordotdev/vector/pull/12730